logger.js

```diff
@@ -58,9 +58,9 @@ module.exports = function(config) {
 		config.errorLevel = errorLevel;
 	};
 
-	function snipsecret(data) {
+	function snipFragileData(data) {
 		for (var key in data) {
-			if (key.search(/secret/i) > -1) {
+			if (key.search(/passphrase|password/i) > -1) {
```

Comment:
Why password here?

Reply:
Because `password` is also sensitive data, same as `passphrase`.

Reply:
Same as secret then.

Reply:
But unused.